tracer/test/Datadog.Trace.Tools.Runner.IntegrationTests/Checks/AgentConnectivityCheckTests.cs

```diff
@@ -91,7 +90,7 @@ public async Task DetectTransportUds()
 
             _ = await AgentConnectivityCheck.RunAsync(processInfo!);
 
-            console.Output.Should().Contain(ConnectToEndpointFormat(url, "domain sockets"));
+            console.Output.Should().Contain(ConnectToEndpointFormat(uri.PathAndQuery, "domain sockets"));
```

Comment:
No, pretty sure this is wrong. It should contain `tracesUdsPath` at least, but I don't see why it shouldn't remain the same with the `unix://` prefix tbh

Reply:
As discussed together, relative paths aren't supported so I'll keep using PathAndQuery but I'll make sure to log proper warnings